hack/machineset.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
```

Comment:
we probably want to `set -euo pipefail` here.
See also http://redsymbol.net/articles/unofficial-bash-strict-mode/